pkg/apis/credentialprovider/install/BUILD

```diff
@@ -0,0 +1,29 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["install.go"],
+    importmap = "k8s.io/cloud-provider-gcp/cmd/auth-provider-gcp/apis/credentialprovider/install",
+    importpath = "k8s.io/kubelet/pkg/apis/credentialprovider/install",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//staging/src/k8s.io/apimachinery/pkg/runtime:go_default_library",
```

Comment:
Staging seems wrong.

Reply:
Fixed - not sure how that slipped by me, my apologies.